src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue23575.cs

```diff
@@ -0,0 +1,23 @@
+﻿using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues
+{
+	public class Issue23575 : _IssuesUITest
+	{
+		public Issue23575(TestDevice testDevice) : base(testDevice)
+		{
+		}
+
+		public override string Issue => "Gradient never returns to the correct colour";
+
+		[Test]
+		[Category(UITestCategories.Border)]
+		public void LinearGradientShouldHaveTheSameTopColorAsBackgroind()
+		{
+			App.WaitForElement("Label");
+			VerifyScreenshot();
```

Comment:
Snapshots available int he latest build 
![image](https://github.com/user-attachments/assets/02192e01-aa81-4a2c-984b-395208bfcffa)
Could you commit the images?

Reply:
Added :)